### DISCUSSION TITLE
Add FLUX for lora's trainings / Finetuning

### DISCUSSION THREAD

**Discussion by BenDes21** [bmaltais/kohya_ss#2699] [Ideas]
@bmaltais  Since the training scripts has been released : https://github.com/XLabs-AI/x-flux

Wondering if we can implement it into kohya_ss 

Thanks a lot

**Comment by bmaltais** [comment]
Konya had been poking at it… not sure when he will have a viable trainer. In the mean time you can try SimpleTrainer

**Reply by BenDes21** [reply]
Alright thanks you

**Reply by BenDes21** [reply]
> Konya had been poking at it… not sure when he will have a viable trainer. In the mean time you can try SimpleTrainer

@bmaltais https://github.com/kohya-ss/sd-scripts/pull/1374/files/da4d0fe0165b3e0143c237de8cf307d53a9de45a..36b2e6fc288c57f496a061e4d638f5641c32c9ea

**Comment by Weatherby43** [comment]
Kohya_ss also has the scripts in a different t branch 

https://github.com/kohya-ss/sd-scripts/tree/flux

**Comment by bmaltais** [comment]
Still only contain SD3 code… but there is hope.

**Reply by BenDes21** [reply]
> Still only contain SD3 code… but there is hope.

flux_train_network.py added

**Reply by gangisbrawn** [reply]
so how do i train it in there how do i input the new code into kohya. the flux supported version?
